### PR TITLE
Add Raspberry Pi installation guide using windows

### PR DESCRIPTION
**Pull request by olbjo**

### CODE REVIEW

File: doc/prepare_raspberrypi.rst [hunk 1]
```diff
@@ -1,5 +1,22 @@
-Install Raspberry Pi OS on the Raspberry Pi (by using a Linux laptop)
-=====================================================================
+Installation and configuration of Raspberry Pi
+==============================================
+The p-net stack and sample application
+has been tested with:
+* Raspberry Pi 3 Model B+
+
+To avoid problems it is recomended to start with a fresh
```
Comment (pyhys) [inline doc/prepare_raspberrypi.rst:7]: recommended

File: doc/prepare_raspberrypi.rst [hunk 2]
```diff
@@ -53,6 +70,66 @@ Log in to it::
...
+* Start Raspberry Pi Imager
+* In the Select OS dialog choose full version
+* Select SD-card
+* Press Write
+
+Step 2. Initial configuration of Raspberry Pi OS.
+
+* Eject SD-card 
+* Reinsert SD-card in windows PC. The SD-card will be shown as external drive named "boot".
+* Enable ssh by creating an empty file named ''ssh'' in the root folder of "boot".
```
Comment (pyhys) [inline doc/prepare_raspberrypi.rst:92]: Use double back-ticks for inline code examples. (The example four lines below works fine)

File: doc/prepare_raspberrypi.rst [hunk 3]
```diff
@@ -53,6 +70,66 @@ Log in to it::
...
+* Insert SD-card and power on Raspberry Pi.
+* Login (preferably using serial console) with default user ''pi'' and password ''raspberry''.
+
+Step 4. Network configuration.
+
+Use the nano editor to edit the configuration files as described below. 
+For example to edit the ''/etc/dhcpcd.conf''::
+
+    sudo nano /etc/dhcpcd.conf
+
```
Comment (pyhys) [inline doc/prepare_raspberrypi.rst:112]: `Save the file in nano by pressing ``CTRL-X``, then ``Y`` and ``Enter``.`

**Review by pyhys — COMMENTED** [review]
Great! 
Just minimal nitpick, see below.

File: doc/prepare_raspberrypi.rst (hunk 1 above)
Comment (olbjo) [reply doc/prepare_raspberrypi.rst:7]: Done, also enabled spelling check for .rst files..

File: doc/prepare_raspberrypi.rst (hunk 2 above)
Comment (olbjo) [reply doc/prepare_raspberrypi.rst:92]: Done, I think I need new glasses.

File: doc/prepare_raspberrypi.rst (hunk 3 above)
Comment (olbjo) [reply doc/prepare_raspberrypi.rst:112]: Done

**Comment by olbjo** [comment]
All comments fixed

**Review by pyhys — APPROVED** [review]

**Pull request merged**